model/task/db.go

```diff
@@ -2203,7 +2203,7 @@ type GroupedTaskStatusCount struct {
 	StatusCounts []*StatusCount `bson:"status_counts"`
 }
 
-func GetTaskStatsByVersion(versionID string, opts GetTasksByVersionOptions) (*TaskStats, error) {
+func GetTaskStatsByVersion(ctx context.Context, versionID string, opts GetTasksByVersionOptions) (*TaskStats, error) {
```

Comment:
It looks like this function is still using the wrapped Aggregation and not passing the context through to the driver.

Reply:
Good catch!